src/lib/snyk-to-html.ts

```diff
@@ -77,6 +78,16 @@ class SnykToHtml {
               ? path.join(__dirname, '../../template/iac/test-report.hbs')
               : template;
           return processIacData(data, template, summary);
+        } else if (
+          data?.runs ||
+          data[0]?.runs
+          //data.runs[0].tool.driver.name == "SnykCode"
+        ) {
+          template =
+          template === path.join(__dirname, '../../template/test-report.hbs')
+            ? path.join(__dirname, '../../template/code/test-report.hbs')
+            : template;
```

Comment:
I understand its probably better in processCodeData but I wanted to keep it consistent. Please let me know

Reply:
I'll leave that decision to you. I won't block this PR on it, it can done in a separate PR. And yes, for consistency, the IaC line will need to be migrated too.

I think it's worthwhile to help draw team boundaries and reduce future cross-team code reviews which are slow and disruptive. Hammer may have a stronger opinion on it in future PRs depending on how often this triggers a review.

Reply:
For now, I want to keep it the same to minimize the changes